src/Query/User/Model/ScanResult.php

```diff
@@ -61,6 +61,6 @@ public function isNotAvailable(): bool
      */
     public function content(): array
     {
-        return json_decode($this->content, true);
+        return json_decode($this->content, true, 512, JSON_THROW_ON_ERROR);
```

Comment:
Again, I'd suggest to just not enforce throw flag.

I am aware that an exception thrown in that place, no matter if by decode returning false or throwing an error, should actually never happen and when it does the exception class is the slightest problem.